financial-connections/src/main/java/com/stripe/android/financialconnections/ui/components/TextField.kt

```diff
@@ -0,0 +1,46 @@
+@file:OptIn(ExperimentalMaterialApi::class)
+
+package com.stripe.android.financialconnections.ui.components
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.ExposedDropdownMenuDefaults.outlinedTextFieldColors
+import androidx.compose.material.OutlinedTextField
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import com.stripe.android.financialconnections.ui.theme.FinancialConnectionsTheme
+
+@Composable
+internal fun FinancialConnectionsOutlinedTextField(
```

Comment:
Is it not possible to reuse `TextFieldUI`?

Reply:
I'd like to see a `payments-ui-core' and maybe a `stripe-ui-core`. Do let us know if you feel like you're starting to see a lot of common components. No need to do that for this PR or workflow tho.